test/Common/Fakes/fake-grpc-error.ts

```diff
@@ -1,15 +1,17 @@
 /**
  * Convenience class for creating more specific Error objects to mimick gRPC error responses.
  */
-// TODO(amiecorso): merge xrp and ilp fake-grpc-errors into common location
 export default class FakeGRPCError extends Error {
   /**
    * Construct a new FakeServiceError.
    *
    * @param message The text details of the error.
    * @param code The gRPC status code.
    */
-  constructor(public readonly message: string, public readonly code: number) {
+  constructor(
+    public readonly code: number,
+    public readonly message: string = '',
```

Comment:
This doesn't really matter, but I believe message is a property on the super class so when you make this `public readonly` it actually overrides that member. 

It's probably best to remove `public readonly` just in case it causes unexpected behavior in the future.